src/status_im/utils/ethereum/abi_spec.cljs

```diff
@@ -96,9 +96,7 @@
         encoded-value  (if encoded-value?
                          (subs value 2)
                          (from-utf8 value))]
-    (str (when dynamic? (enc {:type :int :value (if encoded-value?
-                                                  (count encoded-value)
-                                                  (/ (count encoded-value) 2))}))
+    (str (when dynamic? (enc {:type :int :value (/ (count encoded-value) 2)}))
```

Comment:
Do you know why it was done this way before?

Reply:
no, that's why Eric in reviewers, maybe he could give an answer

Reply:
I'm not sure, add some tests for it if it works better now